### PR TITLE
Fix screen flicker on page scrolling, make navigation bar permanently fixed on top

### PR DESCRIPTION
**Pull request by binarymaster**
Should fix [ONLINE-885](https://jira.reactos.org/browse/ONLINE-885).

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @Fraizeraust and @foxlet for testing: https://pr36.web-content.reactos.org/

**Comment by Fraizeraust** [comment]
The patch you proposed fixes the issue on the Download page, at least on my screen resolution -- **1920 x 1080**.

**Comment by binarymaster** [comment]
I came up with CSS only solution based on this answer: https://stackoverflow.com/a/33788333/4381493

Please review. :)

File: themes/reactos/layouts/partials/nav.html [hunk 1]
```diff
@@ -1,57 +1,4 @@
...
-					</div>
-				</form>
-			</div>
-		</div>
-	</div>
-</div>
+<!-- Padding only -->
+{{ partial "navbar.html" . }}
+<!-- Actual navigation bar -->
+{{ partial "navbar.html" . }}
```
Comment (learn-more) [inline themes/reactos/layouts/partials/nav.html]: Include the same file twice?

Comment (binarymaster) [reply themes/reactos/layouts/partials/nav.html]: Exactly. It's being distinguished by CSS `.navbar-affixed-top:nth-child(2)`.

**Pull request merged**